app/views/ReportView/components/AnalystComments/__tests__/AnalystComments.test.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { when, resetAllWhenMocks } from 'jest-when';
+
+import { withReportContext } from '@/test/testHelpers';
+import withLoading from '@/hoc/WithLoading';
+import { ReportType } from '@/context/ReportContext';
+import api from '@/services/api';
+import AnalystComments from '..';
+
+const mockReport = {
+  ident: '085afd-957bba-55acfd',
+} as ReportType;
+
+const mockComments = {
+  comments: '<img></img><p>test</p>',
+};
+
+const mockComments2 = {
+  comments: '<div style="color: red;">test</div>',
+};
+
+jest.mock('@/services/api');
+
+describe('AnalystComments', () => {
+  beforeEach(() => resetAllWhenMocks())
+
+  test('Img tags are sanitized', async () => {
+    when(api.get)
+      .mockImplementation(() => ({ request: async () => null }))
```

Comment:
TS be screaming at you with the wrong typedefs, else this works 

I guess you can cheat with 
`when(api.get as (endpoint: string) => Partial<ApiCall>)`

Reply:
That works